src/main/java/seedu/duke/helper/UI.java

```diff
@@ -39,10 +39,28 @@ public void printGreeting() {
     public void printHelp() {
         printNewLineSeparator();
         System.out.println("Here are the commands and examples:");
-        System.out.println("1. add patient\n2. delete patient\n3. view patient");
-        System.out.println("4. add doctor\n5. delete doctor\n6. view doctor");
-        System.out.println("6. add patient\n7. delete patient\n9. view patient");
-        System.out.println("10. bye");
+        System.out.println("1. add patient /info. Example:");
+        System.out.println("add patient /info S1234567A, John Doe, 23, M, 10 Baker Street,"
+                + " 1999-12-31, 2021-02-15");
```

Comment:
perhaps just use "\n " and concatenate instead of multiple printlines

Reply:
I can but it will be really messy, especially next time when we add way more help statements so I'll keep it for now